orte/util/nidmap.c

```diff
@@ -296,7 +303,46 @@ int orte_util_nidmap_create(opal_pointer_array_t *pool, char **regex)
             if (startnum < 0) {
                 prefix[j++] = node[i];
             }
+            else {
```

Comment:
The comment does not look right to me.
I think that would `tryreverse` even with ‘x0y’